Shell/Tests/loop.sh

```diff
@@ -42,9 +46,32 @@ for ((test 1 = 1) (test 2 = 2)) {
     $it || unset implicit_ok
 }
 
+# Infinite loop
+for {
+    infinite_ok=yes
+    break
+    unset break_ok
+}
```

Comment:
I'm not 100% on this, `for { ... }` sounds...dumb...
but adding another keyword for almost zero gain sounds even more dumb - and I am therefore conflicted.
Ideas, anyone?

Reply:
\<bugaevc\> how about `loop { }` \</bugaevc\>

Reply:
We discussed this in IRC, actually
I personally prefer `loop` too, but an extra keyword just for an infinite loop doesn't sound convincing.
Though I suppose we could use `loop` in place of `while`? I'm also not that interested in `while`, it seems to always be used for nefarious purposes (`whatever | while read...`  which could be better expressed as `for $(whatever) { match $it ... }` here to the same effect)

Reply:
`loop` it is.